Dockerfile

```
@@ -0,0 +1,6 @@
+FROM python:3-slim
+WORKDIR /app
+COPY . .
+RUN pip install poetry
+RUN poetry install --no-dev
```

```suggestion
RUN poetry install --no-dev --extras "js"
```
Should probably include all optional components in the docker build, which for now is just the "js" feature.